ogr/ogrsf_frmts/gml/ogrgmldriver.cpp

```diff
@@ -196,6 +196,11 @@ void RegisterOGRGML()
         "    <Value>YES</Value>"
         "    <Value>NO</Value>"
         "  </Option>"
+        "  <Option name='OGR_SCHEMA' type='string' description='"
+        "   Partially or totally overrides the auto-detected schema to use for "
+        "creating the layer."
+        "   The overrides are defined as a JSON list of field definitions."
+        "   This can be a filename or a JSON string or a URL.'/>"
```

Comment:
```suggestion
        "Partially or totally overrides the auto-detected schema to use for "
        "creating the layer. "
        "The overrides are defined as a JSON list of field definitions. "
        "This can be a filename or a JSON string or a URL.'/>"
```